backend/app/db/models.py

```diff
@@ -343,7 +343,7 @@ class Event(Base):  # noqa: D101
     )
     name = sa.Column(sa.Text, nullable=False)
     description = sa.Column(sa.Text, nullable=False)
-    created_ts = sa.Column(sa.DateTime(timezone=True), server_default=func.now())
```

Comment:
I thought we also decided to add an explicit year (or even date) field to the `Document` model too?

Reply:
Yes, after merging your currently open PR?

Reply:
Yes, after merging your currently open PR?